src/main/resources/META-INF/plugin.xml

```diff
@@ -101,7 +101,7 @@
     <!-- Android Studio 2.4 preview 1-->
     <!-- see http://tools.android.com/build/studio/idea-tags -->
     <!-- see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html -->
-    <idea-version since-build="171"/>
+    <idea-version since-build="162.376"/>
```

Comment:
Same thing here, we want to keep it at 171 so that it only shows up on android studio 3.0+